api/ws_api.go

```diff
@@ -189,18 +189,27 @@ func (s *Server) manageWsConn(ws *websocket.Conn) {
 
 		case md.CodeAttack:
 			req := NewRequest(s, nil, payload)
+			// response will have the IsTurn field of attacker
 			resp, defender := req.HandleAttack()
 
+			// attacker turn is set to false
+			resp.Payload.IsTurn = false
 			if err := ws.WriteJSON(resp); err != nil {
 				log.Println(err)
 				continue
 			}
 
+			// defender turn is set to true
+			resp.Payload.IsTurn = true 
+			if err := defender.WsConn.WriteJSON(resp); err != nil {
+				log.Println(err)
+				continue
+			}
+
```

Comment:
can we use `SendMsgToBothPlayers` function here?

Reply:
I thought about it. In that case we need to return game too... I mean we can do that I don't know.

Reply:
I'm writing tests for the attack. I'll apply this change (`SendMsgToBothPlayers`) in that PR.